### PR TITLE
Review comments by Russ

### PR DESCRIPTION
**Pull request by hannestschofenig**

### CODE REVIEW

File: draft-ietf-teep-architecture.md [hunk 1]
```diff
@@ -639,9 +639,8 @@ the device can vary. The variations depend on whether the Untrusted Application
 together or are provided separately, and this has implications to the management of
 the TAs in a TEE. In addition to the Untrusted Application and TA(s), the TA(s) and/or TEE may also require additional data to personalize the TA to the device or a user.
 Implementations must support encryption of such
-Personalization Data to preserve the confidentiality of potentially
-sensitive data contained within it, and must support integrity protection
-of the Personalization Data.
+Personalization Data to preserve the confidentiality and integrity protection of potentially
+sensitive data contained within it.
```
Comment (mingpeiwk) [inline draft-ietf-teep-architecture.md:643]: How about:

"Implementation must support mechanisms for the confidentiality such as encryption and integrity protection of such Personalization Data."

or
"Implementation must support encryption for confidentiality and integrity protection of such Personalized Data."

The current revision seems to imply "integration protection" over only potentially sensitive data, not the Personalization Data entirely.

**Review by mingpeiwk — APPROVED** [review]

File: draft-ietf-teep-architecture.md [hunk 2]
```diff
@@ -354,9 +354,9 @@ reducing access to keying material.
 ## Internet of Things
 
 Weak security in Internet of Things (IoT) devices has been posing threats to 
-critical infrastructure that relies upon such devices.
-It is desirable that IoT devices can prevent malware from
-manipulating actuators (e.g., unlocking a door), or
+critical infrastructure, i.e. assets that are essential for the functioning
```
Comment (dthaler) [inline draft-ietf-teep-architecture.md:357]: i.e. should always have a comma following it, per Chicago Manual of Style (which the RFC editor follows)

File: draft-ietf-teep-architecture.md [hunk 3]
```diff
@@ -639,9 +639,8 @@ the device can vary. The variations depend on whether the Untrusted Application
 together or are provided separately, and this has implications to the management of
 the TAs in a TEE. In addition to the Untrusted Application and TA(s), the TA(s) and/or TEE may also require additional data to personalize the TA to the device or a user.
 Implementations must support encryption of such
-Personalization Data to preserve the confidentiality of potentially
-sensitive data contained within it, and must support integrity protection
-of the Personalization Data.
+Personalization Data to preserve the confidentiality and integrity protection of potentially
```
Comment (dthaler) [inline draft-ietf-teep-architecture.md:642]: Delete "protection".
You preserve the integrity, you don't preserve protection per se.

File: draft-ietf-teep-architecture.md [hunk 4]
```diff
@@ -1260,7 +1259,7 @@ against the TEE.
 
 The compromised REE
 may terminate the TEEP Broker such that TEEP transactions cannot reach the TEE,
-or might drop or delay messages between a TAM and a TEEP Agent.
+or might drop, replay or delay messages between a TAM and a TEEP Agent.
```
Comment (dthaler) [inline draft-ietf-teep-architecture.md:1262]: Add oxford comma for consistency with rest of doc

File: draft-ietf-teep-architecture.md [hunk 5]
```diff
@@ -1288,8 +1287,7 @@ for protecting the resource usage allocated for Trusted Component management.
 ## CA Compromise or Expiry of CA Certificate
 
 A root CA for TAM certificates might get compromised or its certificate might
-expire, or a Trust Anchor other than a root CA certificate may also expire or
-be compromised.
+expire.
```
Comment (dthaler) [inline draft-ietf-teep-architecture.md:1290]: I think this change is incorrect.  Line 1287 is about expiry of any CA, not just a root CA.   The change limits it to only root CA and contradicts the section title.

Comment (mingpeiwk) [reply draft-ietf-teep-architecture.md:1290]: Restored in my version.

File: draft-ietf-teep-architecture.md (hunk 4 above)
Comment (mingpeiwk) [reply draft-ietf-teep-architecture.md:1262]: Fixed in a new PR 238.

File: draft-ietf-teep-architecture.md (hunk 3 above)
Comment (mingpeiwk) [reply draft-ietf-teep-architecture.md:642]: Fixed in a new PR 238.

File: draft-ietf-teep-architecture.md (hunk 2 above)
Comment (mingpeiwk) [reply draft-ietf-teep-architecture.md:357]: Fixed in a new PR 238.

**Pull request merged**